<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayForm</class>
 <widget class="QWidget" name="DisplayForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>649</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTableWidget" name="tableWidgetDisplayInfo">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>621</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonPersonSort">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>People</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonComputerSort">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Computers</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConnectionsSort">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Connections</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonAscending">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Ascending</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButtonDescending">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>95</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Descending </string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxSortPerson">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>330</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Age</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxSortComputer">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Name</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBoxSortConnection">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>330</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Person Name</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Computer Name</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
